.github/workflows/build.yml

```diff
@@ -557,10 +634,13 @@ jobs:
       contents: write
       id-token: write
     needs:
-      - unit-test-3-8
-      - unit-test-3-9
-      - unit-test-3-10
-      - unit-test-3-11
+      - metadata
+      - pytest-linux-3-8-st
+      - pytest-linux-3-9-mt
+      - pytest-linux-3-10-mt
+      - pytest-linux-3-11-mt
+      - pytest-windows-3-11-mt
+      - pytest-macos-3-11-mt
```

Comment:
These jobs only run on PRs, so this won't work when we actually go to do a release.